nuxeo-user-group-management/nuxeo-view-user.html

```diff
@@ -29,7 +29,7 @@
 -->
 <dom-module id="nuxeo-view-user">
   <template>
-    <style>
+    <style include="iron-flex nuxeo-styles">
```

Comment:
At some point we need to remove nuxeo-styles from UI elements

Reply:
https://jira.nuxeo.com/browse/ELEMENTS-494